### PR TITLE
Remove old back end and containerise new back end

### PR DESCRIPTION
**Pull request by Zimbardoex**
#### Solves:
- #234 

#### Additional notes:
The new back-end still needs setup for Travis

[Code review checklist](https://github.com/op-analytics/NYT-Media-Analytics/wiki/Code-Review-Checklist)

### CODE REVIEW

**Comment by NICHTJ3** [comment]
.travis.yml needs updated

**Comment by Zimbardoex** [comment]
> .travis.yml needs updated

Done, removed back-end test entirely

File: docker-compose-prod.yml [hunk 1]
```diff
@@ -9,23 +9,24 @@ services:
     volumes:
       - 'nyta-db:/data/db'
     ports:
-      - 27017:27017
+      - 5555:27017
```
Comment (NICHTJ3) [inline docker-compose-prod.yml]: This needs changed back for production :smile:

File: express-back-end/.dockerignore [hunk 2]
```diff
@@ -1,6 +1,8 @@
+node_modules
+coverage
+build
 env
+htmlcov
```
Comment (NICHTJ3) [inline express-back-end/.dockerignore]: Htmlcov and coverage?

File: nginx/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
 FROM nginx:1.15.9-alpine
 
 RUN rm /etc/nginx/conf.d/default.conf
-COPY /dev.conf /etc/nginx/conf.d
+COPY /nginx.conf /etc/nginx/conf.d
```
Comment (NICHTJ3) [inline nginx/Dockerfile]: Thanks for this. Not really needed anymore

File: nginx/nginx.conf [hunk 4]
```diff
@@ -0,0 +1,19 @@
+server {
+  listen 80;
+  proxy_pass_request_body on;
+  proxy_set_header  X-Forwarded-For $remote_addr;
+
```
Comment (NICHTJ3) [inline nginx/nginx.conf]: Thanks for fixing the issue with the trailing spaces :dancer:

File: docker-compose-prod.yml (hunk 1 above)
Comment (Zimbardoex) [reply docker-compose-prod.yml]: Yup, it sure does buddy. Done!

File: express-back-end/.dockerignore (hunk 2 above)
Comment (Zimbardoex) [reply express-back-end/.dockerignore]: Copy-paste gone wrong :woman_shrugging:. Fixed

**Pull request merged**